includes/js_defaults.inc.php

```diff
@@ -115,6 +115,15 @@
 settings["unmarked_image"] =               "<?php echo $theme_config['unmarked_image']; ?>";
 <?php endif; ?>
 
+var user_settings = new Array();
+<?php if (isset ($usersettings['browser_window_target']) and $usersettings['browser_window_target'] == 1): ?>
+user_settings["open_links_in_new_window"] = "EXTERNAL";
+<?php elseif (isset ($usersettings['browser_window_target']) and $usersettings['browser_window_target'] == 2): ?>
+user_settings["open_links_in_new_window"] = "ALL";
+<?php else: ?>
+user_settings["open_links_in_new_window"] = "NON";
```

Comment:
Please note, there was a type, i.e. NON is replaced/corrected by NONE.

Reply:
I thought, this could be the case. :-) But on the other hand `NON` could also be a similar abbrevation like `NAN` in JS for "not a number". I will correct it and add the case `DEFAULT`.

Reply:
Thank you ;-)